src/ask.js

```diff
@@ -20,16 +21,22 @@ exports.handler = (event, context, callback) => {
       body: JSON.stringify(body),
     })
 
-  axios
-    .get(`https://api.stackexchange.com/search/advanced?${params}`)
-    .then(({ data }) => {
-      respond({
+  // if the text value is null just send a response skip the api call.
+  text === null
```

Comment:
Yeah so here I would recommend just using an if statement instead of a ternary.

```
if (!text) {
  respond(...)
} else {
  axios(...)
}
```

Reply:
ok will do